codalab/lib/bundle_cli.py

```diff
@@ -3725,12 +3725,25 @@ def do_chown_command(self, args):
             Commands.Argument(
                 '-d', '--disk-quota', help='Total amount of disk allowed (e.g., 3, 3k, 3m, 3g, 3t)'
             ),
+            Commands.Argument(
```

Comment:
Can you add a page / guide in the documentation about:
- what protected mode is
- how to set up a server with protected mode
- what commands on the CLI need to be run to grant / remove access

Reply:
I added a section to `CLI-Reference.md`. Can you take a look at it?